server/athenian/api/controllers/contributors_controller.py

```diff
@@ -43,7 +43,7 @@ async def get_contributors(request: AthenianWebRequest, id: int) -> web.Response
         prefixer = await Prefixer.load(meta_ids, request.mdb, request.cache)
         settings = Settings.from_request(request, account_id)
         release_settings, logical_settings = await gather(
-            settings.list_release_matches(repos),
+            settings.list_release_matches(prefixer, repos),
             settings.list_logical_repositories(prefixer, repos),
```

Comment:
Since both of the `list` methods require the prefixer now, we should move it to the `Settings.__init__`